src/Ocelot/Configuration/SecurityOptions.cs

```diff
@@ -1,17 +1,43 @@
 ﻿using System.Collections.Generic;
+using System.Linq;
+using NetTools;
 
 namespace Ocelot.Configuration
 {
     public class SecurityOptions
     {
-        public SecurityOptions(List<string> allowedList, List<string> blockedList)
+        public SecurityOptions(List<string> allowedList, List<string> blockedList, bool excludeAllowedFromBlocked)
         {
-            IPAllowedList = allowedList;
-            IPBlockedList = blockedList;
+            IPAllowedList = new List<string>();
+            IPBlockedList = new List<string>();
+            ExcludeAllowedFromBlocked = excludeAllowedFromBlocked;
+
+            foreach (var allowed in allowedList)
+            {
+                if (IPAddressRange.TryParse(allowed, out var allowedIpAddressRange))
+                {
+                    var allowedIps = allowedIpAddressRange.AsEnumerable().Select(x => x.ToString());
+                    IPAllowedList.AddRange(allowedIps);
```

Comment:
In case of `192.168.1.0/255.255.255.0` subnet,
Will we have 255 elements in the list with values **192.168.1.x** ?

Reply:
Sure, here a screenshot from LINQPad example of this case
![image](https://github.com/ThreeMammals/Ocelot/assets/6802895/139be9a1-c8d4-443b-a5c4-89c2ae9c151f)

Reply:
🆗  Not an issue!